blog/2023-06-20-newsletter.mdx

```diff
@@ -0,0 +1,137 @@
+* [Conda survey results](https://conda.org/blog/2022-03-30-conda-survey/), blog post by Dave Clements summarizing last year's survey of conda users, integrated with relevant bits from the PyPA's survey
+* The [Conda Community Forum](https://conda.discourse.group/) saw navigation improvements with a new sidebar for navigating the site and a new top menu for navigating to other conda websites.
+
+
+## Conda, now starring on YouTube
+
+**Conda now has a [YouTube channel](https://www.youtube.com/channel/UCNc3BL-gO8hbx_7DdEDcofg)** with four playlists/videos to start you off: **[Packaging](https://youtu.be/6by8b2Ht200), [Presentations](https://youtu.be/OeSv0_HesaU), [Training](https://youtu.be/NSgTFf40vHQ)**, and **[Releases](https://youtu.be/VD5TXcbYgR4)**.
+
+<Image img={youtube} alt="Montage showing screenshots from 4 videos."/>
+
+Or, a long version.  I prefer the short one.
+
+* [Packaging](https://youtu.be/6by8b2Ht200), featuring *Python Distribution & Packaging Roundtable*, An OpenTeams panel, May 2023.
+* [Presentations](https://youtu.be/OeSv0_HesaU), featuring *Plug life into your codebase: Making an established Python codebase pluggable*, a talk by Bianca Henderson at PyCon US 2023, April 2023.
+* [Training](https://youtu.be/NSgTFf40vHQ): featuring *Publishing your Python project, the conda way*, a tutorial by Valerio Maggio, Mahe Iram Khan, Bianca Henderson (again!), and Dave Clements, at PyCon US 2023, April 2023.
+* [Releases](https://youtu.be/VD5TXcbYgR4): featuring the *Conda 22.11 Release Video*, by the contributors, December 2023.
+
+
+## Releases and Announcements
+
+* **[conda-lock 2.0.0](https://github.com/conda/conda-lock/releases/tag/v2.0.0)** - a lightweight library for generating fully reproducible lock files for conda environments.
+* **[conda 23.5.0](https://conda.org/blog/2023-05-31-may-2023-releases#changes-in-conda)** - Python 3.11 support and the new [**conda doctor** subcommand plugin](https://conda.org/blog/2023-06-01-conda-doctor) for detecting corrupted installs.
+* **[conda-build 3.25.0](https://conda.org/blog/2023-05-31-may-2023-releases#changes-in-conda-build)** - an update to `conda`'s package builder.
+* **[conda-libmamba-solver 23.5.0](https://conda.org/blog/2023-05-31-may-2023-releases#changes-in-conda-libmamba-solver)** - An update to the fast `libmamba` solver in `conda`.
+* **[grayskull 2.3.1](https://github.com/conda/grayskull/releases/tag/v2.3.1)** - An automatic recipe generator for the conda ecosystem.
+* **[Prefix.dev Channels](https://prefix.dev/blog/introducing_channels)** - A new option for creating your own conda channels.
+* **[conda-package-streaming 0.8.0 & conda-package-handling 2.1.0](https://conda.discourse.group/t/conda-package-streaming-0-8-0-and-conda-package-handling-2-1-0-released/267)** - read new & old formats (`.conda & .tar.bz2`) and create & extract conda packages.
```

Comment:
```suggestion
* **[conda-package-streaming 0.8.0 & conda-package-handling 2.1.0](https://conda.discourse.group/t/conda-package-streaming-0-8-0-and-conda-package-handling-2-1-0-released/267)** - Read new & old formats (`.conda & .tar.bz2`) and create & extract conda packages.
```
This is also the only item in this list whose explanation starts with a verb. Maybe "Updates for reading new and old formats ... and creating and extracting conda packages."?